api.go

```diff
@@ -358,6 +365,28 @@ func getParam(param string, r *http.Request) (string, error) {
 	return strings.ToLower(unicode), nil
 }
 
+// Retrieves `param` as a query parameter from `http.Request` r, and tries to cast it as
+// a number between [lowInc, highExc). If fails, then returns an error.
+// If `param` isn't specified, return defaultNum.
+func getInt(param string, r *http.Request, lowInc int, highExc int, defaultNum int) (int, error) {
+	unicode := r.FormValue(param)
+	if unicode == "" {
+		return defaultNum, nil
+	}
+	n64, err := strconv.ParseInt(unicode, 10, 8)
+	n := int(n64) // best console
```

Comment:
nit - I think we could use https://golang.org/pkg/strconv/#Atoi

signed,
your colleague who only knows how to play tetris :)